app.js

```diff
@@ -0,0 +1,72 @@
+const formContainer = document.getElementById('formContainer');
+const bookTitle = document.getElementById('title');
+const bookAuthor = document.getElementById('author');
+const dynamicBooksDiv = document.getElementById('books-wrapper');
+
+const books = JSON.parse(localStorage.getItem('booksDetails')) || [];
+
+const addBook = (title, author, id) => {
+  if (title && author) {
+    books.push({
```

Comment:
- [ ] OPTIONAL
It will be a good practice to favor the use of array methods such as `concat` over `push`, which doesn't mutate the array.